cmd/manager/main.go

```diff
@@ -57,10 +69,15 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{Namespace: namespace})
+	// Create a new Cmd to provide shared dependencies and start components at the specified
+	// namespace.
+	mgr, err := manager.New(cfg, manager.Options{
+		Namespace:      namespace,
+		MapperProvider: restmapper.NewDynamicRESTMapper,
+	})
 	if err != nil {
 		log.Fatal(err)
+		os.Exit(1)
```

Comment:
why are we using `os.Exit(1)` here?

Reply:
Sorry, this wouldn't be needed. I will rebase and push the changes.